frontend/src/app/report/report.service.ts

```diff
@@ -32,11 +37,13 @@ ReportService {
   public setAreaDict(report: Report): Map<number, string> {
 
     const areaDict = new Map<number, string>();
-    const index_ids = Object.keys(report.scenarioChanges.areaChanges);
+    if(report.scenarioChanges !== null) {
```

Comment:
It can as of this changeset:
https://github.com/havochvatten/MSP-Symphony/blob/5915cfba272927bc0f45d4807f98817419a8942f/frontend/src/app/data/calculation/calculation.interfaces.ts#L46

Reply:
Ah sorry I must have checked the old file